app/main.py

```diff
@@ -6,9 +8,11 @@
 
 app = Flask(__name__)
 app.register_blueprint(api)
-app.config[
-    "SQLALCHEMY_DATABASE_URI"
-] = "postgresql://newschatbotdevelopment:Wlk8skrHKvZEbM6Gw@database.internal.newschatbot.ceskodigital.net:5432/newschatbotdevelopment"
+db_connstr = os.environ.get(
+    "DB_CONNSTR",
+    "postgresql://newschatbotdevelopment:Wlk8skrHKvZEbM6Gw@database.internal.newschatbot.ceskodigital.net:5432/newschatbotdevelopment",
```

Comment:
To URL by bylo ideální ještě extrahovat ven, jako default kdyžtak dát ten `localhost`?